index.html

```diff
@@ -0,0 +1,77 @@
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <article class="welcome">
+      <p class="hello">"Hello! Sharing World"</p>
+      <p class="main-heading">CREATIVE COMMONS GLOBAL SUMMIT 2015</p>
+      <P class="joyful">A joyful festival where people from over 80 countries who create positive change gather and share their thoughts will be held in Korea in October.</P>
+      <p class="schedule">2022.10.14 (Wed) ~ 17 (Sat)</p>
+      <p class="museum">@National Museum of Korea, Art Center Nabi, etc.</p>
+    </article>
+    <article class="main-program">
+      <h3>Main Program</h3>
+      <hr class="red-main-program">
+      <div class="programs">
+      <div class="program-activity"><img src="img/ponents_icon.png" alt="" class="program-icon"> <span class="program-title">Lecture</span><p class="program-detail">Listen to the speakers from various cooutries about the messeges of sharing and opeinging.</p></span></div>
+      <div class="program-activity"><img src="img/exhibition_icon.png" alt="" class="program-icon"> <span class="program-title">CC Exhibition</span><p class="program-detail">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p></div>
+      <div class="program-activity"><img src="img/forum_icon.png" alt="" class="program-icon"> <span class="program-title">Forum</span><p class="program-detail">Have the time to share your thoughts and opinions with experts for each topic.</p></div>
+      <div class="program-activity"><img src="img/workshop_icon.png" alt="" class="program-icon"> <span class="program-title">Workshop</span><p class="program-detail">Try creating your own work using open source license rather than just watching.</p></div>
+      <div class="program-activity"><img src="img/cc-party.png" alt="" class="program-icon"> <span class="program-title">CC Ignite</span><p class="program-detail">Get opportunities to network with CC affiliate around the world, also after the summit</p></div>
+    </div>
+      <span class="seeWholeProgram"><p>SEE THE WHOLE PROGRAM</p></span>
+      <button class="joinCC">Join CC Global Summit 2015</button>
+      
+    </article>
+    <article class="speakers">
+      <h3 class="speaker-heading">Featured Speakers</h3>
+      <hr class="red-main-program">
+      <div class="speaker-group">
+      </div>
+    </article>
+  </section>
+
+  <div class="partners">
+    <div class="partner-info">
+      <h3 class="partners-title  previous-summit-heading">Partner</h3>
+      <hr class="logo-heading-line">
+    </div>
+    <div class="logo-group">
+      <img src="img/mozilla-logo.png" alt="mozilla logo">
+      <img src="img/google-logo.png" alt="google logo">
+      <img src="img/naver.png" alt="naver logo">
+      <img src="img/orange_logo.png" alt="orange logo">
+      <img src="img/airbnb_logo.png" alt="airbnb logo">
+    </div>
+  </div>
```

Comment:
- **[OPTIONAL] The mobile version of the home page finished with the featured Speakers section, please fix it to  be similar to the original design 👇🏽 

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/211155142-703c180e-d11b-4c1b-ab7e-489b367f676a.png) | ![image](https://user-images.githubusercontent.com/58642949/211155154-3663421f-1a8b-4ff0-8f1e-06a368e218b4.png) |